config/example/away_mission_config.txt

```diff
@@ -15,6 +15,7 @@
 #_maps/map_files/RandomZLevels/clownplanet.dmm
 #_maps/map_files/RandomZLevels/example.dmm
 #_maps/map_files/RandomZLevels/spacebattle.dmm
+#_maps/map_files/RandomZLevels/stationCollision.dmm
```

Comment:
I don't really have any intent on making that map better and in retrospect should not have readded it.